src/Sylius/Bundle/UserBundle/DependencyInjection/Configuration.php

```diff
@@ -103,57 +61,102 @@ private function addResourcesSection(ArrayNodeDefinition $node)
+                                                    ->end()
+                                                ->end()
+                                            ->end()
+                                            ->arrayNode('pin')
+                                                ->addDefaultsIfNotSet()
+                                                ->children()
+                                                    ->integerNode('length')
+                                                        ->defaultValue(4)
+                                                        ->min(1)->max(9)
+                                                    ->end()
+                                                    ->scalarNode('field_name')
+                                                        ->defaultValue('passwordResetToken')
+                                                        ->validate()
+                                                        ->ifTrue(function ($passwordResetToken) { return !is_string($passwordResetToken); })
+                                                            ->thenInvalid('Invalid resetting pin field "%s"')
+                                                        ->end()
+                                                    ->end()
+                                                ->end()
                                             ->end()
                                         ->end()
                                     ->end()
-                                ->end()
-                                ->arrayNode('validation_groups')
-                                    ->addDefaultsIfNotSet()
-                                    ->children()
-                                        ->arrayNode('default')
-                                            ->prototype('scalar')->end()
-                                            ->defaultValue(['sylius'])
+                                    ->arrayNode('verification')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->arrayNode('token')
+                                                ->addDefaultsIfNotSet()
+                                                ->children()
+                                                    ->integerNode('length')
+                                                        ->defaultValue(16)
+                                                        ->min(1)->max(40)
+                                                    ->end()
+                                                    ->scalarNode('field_name')
+                                                        ->defaultValue('emailVerificationToken')
+                                                        ->validate()
+                                                        ->ifTrue(function ($emailVerificationToken) { return !is_string($emailVerificationToken); })
```

Comment:
Yeah, that's right, but nevertheless we're never validating it because it brings much more mess than value.